desktop/app/splash.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+// Checks to see if we've launched before (we touch started.txt). If not, write the file and open a browser
+import {socketRoot} from '../../react-native/react/constants/platform.native.desktop'
```

Comment:
we now put this file by the socket file. We had a discussion @cjb @maxtaco , on startup this location should be setup by the daemon (the leveldb / data folder may not be). Currently these folders are in 2 different places. On linux the socketfile is in an ephemeral folder so this won't work until that's changed. Core ticket created: https://keybase.atlassian.net/browse/CORE-2123